.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,35 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Description
+
+A clear and concise description of the bug or issue.
+
+
+## Environment
+
+**Operating System + Version**: 
+**GPU Type**: 
+**Nvidia Driver Version**: 
+**CUDA Version**: 
+**CUDNN Version**: 
+**Python Version (if applicable)**: 
+**TensorFlow Version (if applicable)**: 
+**PyTorch Version (if applicable)**: 
+**TensorRT Version**: 
+**Baremetal or Container (if container which image + tag)**: 
+
+
+## Steps To Reproduce
+
+Craft a minimal bug report following this guide - https://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
+
+This helps us reproduce the issue you're having and resolve the issue more quickly:
```

Comment:
Request for build command and steps to run the test/usecase. Attach or links to any models or relevant data files.

Reply:
Done